src/lib/y2partitioner/actions/edit_btrfs.rb

```diff
@@ -0,0 +1,72 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2partitioner/actions/base"
+require "y2partitioner/dialogs/btrfs_subvolumes"
+require "y2partitioner/ui_state"
+
+module Y2Partitioner
+  module Actions
+    # Action for editing a BTRFS filesystem, see {Actions::Base}
+    class EditBtrfs < Base
+      # Constructor
+      #
+      # @param filesystem [Y2Storage::Filesystems::Btrfs]
+      def initialize(filesystem)
+        super()
```

Comment:
In submitted changes, you are using both, `super()` and `super`.

Better to choose, and use, only one. Isn't it? :)

Reply:
In this case we need to use `super()` because the parent constructor must be called without parameters.